styles/designs/corn/parts/_exercises-components.scss

```diff
@@ -25,16 +32,6 @@ $Exercise__Container--MaterialSet2: (
 
```

Comment:
I have left the above `ExerciseContainerMaterialSet2` properties hardcoded because it is a specific component which is present only in multicolumn exercises with a specific class and it can't affect other exercises.